drivers/gles3/shaders/feed.glsl

```diff
@@ -0,0 +1,41 @@
+/* clang-format off */
+#version 300 es
+
+#[modes]
+
+mode_default = 
+
+#[specializations]
+
+USE_EXTERNAL_SAMPLER = false
+
+#[vertex]
+
+layout(location = 0) in vec2 vertex_attrib;
+
+out vec2 uv_interp;
+
+
+void main() {    
+	uv_interp = vertex_attrib * 0.5 + 0.5;    
+	gl_Position = vec4(vertex_attrib, 1.0, 1.0);
+}
+
+/* clang-format off */
+#[fragment]
+
+layout(location = 0) out vec4 frag_color;
+in vec2 uv_interp;
+
+/* clang-format on */
+#ifdef USE_EXTERNAL_SAMPLER
+uniform samplerExternalOES sourceFeed; // texunit:0
+#else
+uniform sampler2D sourceFeed; // texunit:1
+#endif
```

Comment:
Is it really using `texunit:0` if it's an external texture and `texunit:1` otherwise? This seems a little odd to me - I would have expected it to use the same texture unit for both.

Reply:
ah yes the comment is wrong, it dates from the commit were I handle both feed and depth map at the same time